components/pipeline-service/production/stone-prd-m01/deploy.yaml

```diff
@@ -1334,7 +1468,7 @@ spec:
             secretKeyRef:
               key: db.name
               name: tekton-results-database
-        image: quay.io/redhat-appstudio/tekton-results-api:bae7851ff584423503af324200f52cd28ca99116
+        image: quay.io/redhat-appstudio/tekton-results-api:ed360eccc021ad5eedf8ea9c0732912ef602b15a
```

Comment:
yep we are allowing a bump of the api server image while we still pin the watcher image at the level with the mem leak (and the performance cheats which we need to live with until the WG approved rewrite of logging to bypass the GRPC calls occurs).

the watcher image is not changed here

looks good